scripts/check_configs.ts

```diff
@@ -37,6 +37,7 @@ interface ConfigJSON {
   disabled_transfer_banner_config?: DisabledTransferBannerConfig;
   announcement_banner: AnnouncementBanner;
   quick_select_deposit_options?: QuickSelectToken[];
+  chain_fee_token_map?: ChainFeeTokenMap;
```

Comment:
Since `chain_fee_token_map` is defined as required on config.schema.json, think you can remove the optional sign:
```typescript
interface ConfigJSON {
  ...
  chain_fee_token_map: ChainFeeTokenMap;
}
```